tripplanner/src/components/Map.jsx

```
@@ -0,0 +1,63 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { loadGoogleMaps } from '../lib/maps';
+
+const Map = ({ fullHeight = false, markers = [] }) => {
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)